Xwt.Gtk.Mac/GtkMacAccessibleBackend.cs

```diff
@@ -83,5 +84,30 @@ internal static INSAccessibility GetNSAccessibilityElement (GLib.Object o)
 				nsa.AccessibilityUrl = new NSUrl (value.AbsoluteUri);
 			}
 		}
+
+		public override void AddChild (object nativeChild)
+		{
+			var nsa = GetNSAccessibilityElement (widget.Accessible);
+			var parent = nsa as NSAccessibilityElement;
+			if (parent != null) {
+				var child = nativeChild as NSAccessibilityElement;
+				if (child == null) {
+					throw new Exception ("nativeChild is not NSAccessibilityElement");
+				}
+				parent.AccessibilityAddChildElement (child);
+			}
+		}
+
+		public override void RemoveChild (object nativeChild)
+		{
+			var nsa = GetNSAccessibilityElement (widget.Accessible);
+			var parent = nsa as NSAccessibilityElement;
+
+			if (parent != null) {
+				var children = parent.AccessibilityChildren.ToList ();
```

Comment:
Can we not create a new array that of `parent.AccessibilityChildren.Length - 1` and skip `nativeChild`?

Reply:
What would happen if `nativeChild` is not in `parent.AccessibilityChildren`?
But maybe we can skip 1 iteration, if we remove `children.Remove ((NSObject)nativeChild);` and just enumerate to an array like:
`parent.AccessibilityChildren = parent.AccessibilityChildren.Where(c => c != nativeChild).ToArray ();`

Reply:
Good point. That works for me.